src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1218,3 +1221,20 @@
     (when (fs/exists? cert)
       (fs/delete cert)
       (log/debug "Deleted certificate for" subject))))
+
+(schema/defn ^:always-validate get-custom-oid-mappings :- (schema/maybe OIDMappings)
+  "Given a path to a custom OID mappings file, return a map of all oids to
+  shortnames"
+  [custom-oid-mapping-file :- (schema/maybe schema/Str)]
+  (when (fs/file? custom-oid-mapping-file)
+    (try
+      (let [oid-mappings (:oid_mapping (yaml/parse-string (slurp custom-oid-mapping-file)))]
+        (into {} (for [[oid names] oid-mappings] [(name oid) (keyword (:shortname names))])))
+      (catch Exception e
```

Comment:
I think it's probably best to just remove this error handling altogether if there's not a specific exception we are trying to catch and report in a better way.  I suspect that if `slurp` or `parse-string` throws an error, that should be good enough for a user to figure out what's up.